crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -431,6 +511,16 @@ fn r#move(
 ) -> superposition::Result<PutResp> {
     use contexts::dsl;
     let req = req.into_inner();
+
+    let ctx_condition = req.context.to_owned().into_inner();
+    let ctx_condition_value = Value::Object(ctx_condition.clone().into());
+    let description = if req.description.is_none() {
+        ensure_description(ctx_condition_value.clone(), conn)?
+    } else {
+        req.description.expect("Description should not be empty")
```

Comment:
I know this will work , but if we can use match expression or abstract the some part without expect or unwrap 
would be great